torch/_dynamo/compiled_autograd.py

```diff
@@ -300,7 +300,7 @@ def runtime_wrapper(compiled_fn, inputs, sizes, hooks):
             try:
                 in_compiled_autograd_region = True
                 for i in runtime_inputs_to_move:
-                    inputs[i] = inputs[i].cuda()
+                    inputs[i] = inputs[i].pin_memory().cuda(non_blocking=True)
```

Comment:
Should we put a numel() limit on this ? The general guidance for pin memory is to avoid overallocating.

Reply:
i guess it doesn't matter rn since every input in runtime_inputs_to_move would have numel=1